### PR TITLE
feat(plugin): oneko Coin Game

### PR DESCRIPTION
**Pull request by 0xGingi**
Fork of oneko.js that adds a simple coin collect game

### CODE REVIEW

File: src/plugins/onekoCoins/index.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    start() {
+        const coinCounterEl = document.createElement('div');
+        coinCounterEl.id = 'coin-counter';
+        coinCounterEl.style.position = 'fixed';
+        coinCounterEl.style.right = '10px';
+        coinCounterEl.style.bottom = '10px';
+        coinCounterEl.textContent = 'Coins: 0';
+
+        document.body.appendChild(coinCounterEl);
+        fetch("https://raw.githubusercontent.com/0xGingi/oneko.js/main/oneko.js")
```
Comment (Cynosphere) [inline src/plugins/onekoCoins/index.ts:42]: Pin to a specific commit, otherwise you could push a malicious update and no one would be the wiser.

**Review by Cynosphere — CHANGES_REQUESTED** [review]

File: src/plugins/onekoCoins/index.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    // Listing adryd here because this literally just evals her script
+    authors: [Devs.Ven, Devs.adryd,
+    {
+        id: 351859727568994314n,
+        name: "0xGingi"
+    },
+    ],
+
+    start() {
+        const coinCounterEl = document.createElement('div');
```
Comment (MrDiamondDog) [inline src/plugins/onekoCoins/index.ts:34]: Don't do don manipulation, use patches to put the text where you want

File: src/plugins/onekoCoins/index.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+import { Devs } from "@utils/constants";
+import definePlugin from "@utils/types";
+
+export default definePlugin({
+    name: "oneko coins",
+    description: "oneko but with coins",
+    // Listing adryd here because this literally just evals her script
+    authors: [Devs.Ven, Devs.adryd,
+    {
```
Comment (MrDiamondDog) [inline src/plugins/onekoCoins/index.ts:27]: Add yourself to devs

**Comment by 0xGingi** [comment]
Thanks, will change all that and additionally add this as an optional option inside the original oneko plugin

**Pull request closed without merging**